src/datastores/handlers/base.js

```diff
@@ -25,6 +25,16 @@ class Settings {
       await this.upsert('externalPlayerCustomArgs', newValue)
     }
 
+    // In FreeTube 0.23.0, the "Enable Theatre Mode by Default" setting was incoporated as an option
+    // of the "Default Viewing Mode" setting. This is a one time migration to preserve users'
+    // Theater Mode preference through this change.
+    const defaultTheatreMode = await db.settings.findOneAsync({ _id: 'defaultTheatreMode' })
+
+    if (defaultTheatreMode?.value) {
+      await this.upsert('defaultViewingMode', 'theatre')
+      await this.upsert('defaultTheatreMode', false)
```

Comment:
Instead of setting it to false and keeping it around it might be easier to just delete it.

```suggestion
      await db.settings.removeAsync({ _id: 'defaultTheatreMode'})
```